src/Microsoft.Docs.LearnValidation/LearnValidationLogger.cs

```diff
@@ -45,7 +70,18 @@ public void Log(LearnErrorLevel errorLevel, LearnErrorCode errorCode, string mes
             }
         }
 
-
+        private string FormatMessage(LearnErrorCode errorCode, object[] message)
+        {
+            if (s_errorMessageMapping.TryGetValue(errorCode, out var formatString))
```

Comment:
consider let it throw, if an entry missing that means we have some bugs to fix.

```suggestion
            s_errorMessageMapping[errorCode]
```